app/vlinsert/jsonline/jsonline.go

```diff
@@ -89,7 +104,7 @@ func readLine(lr *insertutil.LineReader, timeField string, msgFields []string, l
 	defer logstorage.PutJSONParser(p)
 
 	if err := p.ParseLogMessage(line); err != nil {
-		return true, fmt.Errorf("cannot parse json-encoded line: %w; line contents: %q", err, line)
```

Comment:
This change is needed to avoid duplicating error messages like:
`cannot parse json-encoded line: cannot parse json: cannot parse JSON ...`